graphql_compiler/tests/snapshot_tests/test_cost_estimation.py

```diff
@@ -276,13 +241,9 @@ def test_fold(self):
             'Animal': 5,
             'Animal_BornAt': 7,
             'Animal_FedAt': 3,
+            'FeedingEvent': 11
```

Comment:
You seem to be adding test data that doesn't change the tests. Why?

Reply:
I address this in my third bullet point (I didn't mention FeedingEvent, but it's the same reason).
It doesn't affect the test result since it gets cancelled out, but since it still gets used and it's a *required* statistic, we need to add this to our test data.

Reply:
Oh ok got it.